### ISSUE TITLE
[CI] Drone build is failing on release branch because client-python is clone from master instead of release branch

### ISSUE THREAD

**Issue by aHenryJard** [LuatixHQ/opencti#8928]
## Description

![image](https://github.com/user-attachments/assets/90a3a801-eaf7-4c37-aaa4-96e14b24a2a7)

It's a forgotten use case on the build configuration that we have updated for release branch process.

## Environment

Drone

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. { e.g. Run ... }
2. { e.g. Click ... }
3. { e.g. Error ... }

## Expected Output

Green CI

## Actual Output

Red CI
On console log there is a "[CLONE-DEPS] END; Using opencti on branch:master" => should be "[CLONE-DEPS] END; Using opencti on branch:release/6.4.0"

**Comment by lndrtrbn** [comment]
Outdated ?

**Comment by aHenryJard** [comment]
I don't think so. It's just that it's failing only when there is new code on client-python: release that makes opencti: release fail with client-python:master.

**Comment by aHenryJard** [comment]
This is not needed anymore since client python repository migration.
